packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -123,7 +132,7 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
         localVideoViewOptions={localVideoViewOptions}
         remoteVideoViewOptions={remoteVideoViewOptions}
         styles={VideoGalleryStyles}
-        layout="floatingLocalVideo"
+        layout={layoutBasedOnTilePosition}
```

Comment:
actually, here the local video tile is rewiring the entire layout including all remote tiles